templates/dashd.conf.template

```diff
@@ -35,23 +21,47 @@ listen=1
 dnsseed=0
 allowprivatenet=0
 
-printtoconsole=1
+{{? it.network.name === 'testnet'}}
```

Comment:
is there "not equal" operator?

Reply:
It's just plain JS, so the "not equal" operator is `!==`

Reply:
So pity. It looks ugly a bit :(. Actually, you can use JS if conditions: https://github.com/olado/doT/blob/master/examples/advancedsnippet.txt#L41-L43

Reply:
Not sure what bit looks ugly to you? You want me to change all my `{{? <condition>}} <output> {{?}}` syntax to `{{if ({{<condition>}}) { }} <output> {{ } }}` right? Isn't that a **lot** uglier?

Reply:
Actually I meant to change `{{? it.network.name === 'testnet'}}{{??}}output{{?}}` to `{{ if (it.network.name !== 'testnet') { }}output{{ } }}`

Reply:
That syntax hurts my eyes! Can you explain what you mean in chat? My understanding is we need to output `[test]` if the network is set to testnet...

Reply:
Yeah, we defo need. But I don't see it right now there. It's just empty `if` body and only `else` body has output. If you put `[testnet]` to `if` body I'm good with that. My point was that keeping an empty `if` body just to use `else` body is ugly.

Reply:
`testnet=1` and `[test]` are in the `if` body - are we looking at the same code?

Reply:
Hehe not sure. Just trust in your words :)))